src/app/api/routes/devices.py

```diff
@@ -39,3 +37,15 @@ async def delete_device(device_id: int = Path(..., gt=0), _=Security(get_current
 @router.get("/my-devices", response_model=List[DeviceOut])
```

Comment:
```suggestion
@router.get("/my_devices", response_model=List[DeviceOut])
```

Reply:
Following FG comment, I think it makes more sense to have hypens rather than underscores